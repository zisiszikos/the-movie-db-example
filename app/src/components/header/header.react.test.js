import React from 'react';
import Header from './header.react.js';

describe('Header component', () => {

    it('should render properly', () => {

        const header = shallow(<Header />);
        expect(header.is('.header')).toBe(true);
        expect(header.find('.header-title').length).toBe(1);
        expect(header.find('.header-title').text()).toEqual('The Movie DB');
        expect(header).toMatchSnapshot();
    });
});
