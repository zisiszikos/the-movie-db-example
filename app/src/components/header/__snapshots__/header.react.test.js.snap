// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render properly 1`] = `
<div
  className="header"
>
  <div
    className="left"
  >
    <h2
      className="header-title"
    >
      The Movie DB
    </h2>
  </div>
  <div
    className="right"
  >
    <Search />
  </div>
</div>
`;
